frontend/view-edit-lead/lead-home/lead-section.component.tsx

```diff
@@ -26,15 +26,28 @@ export default function LeadSection(props: LeadSectionProps) {
         className="unstyled lead-section-header"
         onClick={toggleExpandAndStore}
       >
-        <h1>{props.title}</h1>
+        <h1>{title}</h1>
+        {metadata && <LeadMetadata metadata={metadata} />}
       </button>
-      {expanded && <div className="lead-section-content">{props.children}</div>}
+      {expanded && <div className="lead-section-content">{children}</div>}
     </div>
   );
 }
 
 type LeadSectionProps = {
   title: string;
+  metadata?: {
+    createdBy: {
```

Comment:
There is an existing type for this - `CUObjectAudit`. It's in `edit-invoice.component.tsx`

```suggestion
    createdBy: CUObjectAudit
```

Reply:
Yes, I considered using that but it's not exactly the same so I wasn't sure if I should reuse it or not.